### PR TITLE
Display zones

### PR DESCRIPTION
**Pull request by Atlante45**
Render zones as a wireframe in edit mode.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: cafaeed82255f12c65985d5eeee828362966c1ae available at: https://gist.github.com/5e7a1be55e91849f3824<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4789.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4789.dmg

File: libraries/render-utils/src/Model.h [hunk 1]
```diff
@@ -415,7 +420,7 @@ class Model : public QObject, public PhysicsEntity {
             IS_DEPTH_ONLY = (1 << IS_DEPTH_ONLY_FLAG),
             IS_SHADOW = (1 << IS_SHADOW_FLAG),
             IS_MIRROR = (1 << IS_MIRROR_FLAG),
-
+            IS_WIREFRAME = (1 << IS_WIREFRAME_FLAG)
```
Comment (samcake) [inline libraries/render-utils/src/Model.h]: Can you add a comma at the end of the last enum

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8997879b0244e90f4d3686437eb29306fe62a125 available at: https://gist.github.com/5e7a1be55e91849f3824<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4789.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4789.dmg

**Comment by samcake** [comment]
Tested on windows, works fine for me
Clement is it ready to merge ?

**Comment by Atlante45** [comment]
Yes, just pushed changes for the comma though (and to quiet the compiler).

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7689837463062151f7bf75109f7d575abd442cd3 available at: https://gist.github.com/5e7a1be55e91849f3824<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4789.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4789.dmg

**Pull request merged**